presentation/src/main/java/me/androidbox/presentation/agenda/viewmodel/AgendaViewModel.kt

```diff
@@ -130,4 +144,29 @@ class AgendaViewModel @Inject constructor(
             }
         }
     }
+
+    private fun syncAgendaItems() {
+        agendaSyncObserver = Observer { workInfo ->
+            if(workInfo.state == WorkInfo.State.SUCCEEDED) {
+                /* Delete the ids from the deleted sync table */
+                viewModelScope.launch {
+                    agendaLocalRepository.deleteEventSyncType(SyncAgendaType.DELETE)
```

Comment:
As agendaLocalRepository.deleteEventSyncType(SyncAgendaType.DELETE) doesn't return any result I can just use launch block

Reply:
An issue with this, is that this code never seems to run.

Reply:
@philipplackner Updated